exopy_hqc_legacy/instruments/drivers/visa/tektro_awg.py

```diff
@@ -726,6 +754,20 @@ def running(self, value):
                                  running method''').format(value), 80)
             raise VisaTypeError(mess)
 
+    @instrument_property
+    @secure_communication()
+    def running(self):
+        """Run state getter method
+
+        """
+        self.clear_output_buffer()
+        run = self.ask_for_values("AWGC:RST?")[0]
```

Comment:
Same here please use query_ascii_values or just query followed by a call to int.